src/xmipp/external/libsvm/svm.cpp

```diff
@@ -2961,6 +3026,9 @@ void svm_free_model_content(svm_model* model_ptr)
 	free(model_ptr->probB);
 	model_ptr->probB= NULL;
 
+	free(model_ptr->sv_indices);
+	model_ptr->sv_indices = NULL;
```

Comment:
Unmodified to avoid having two criteria in the same script: NULL (deprecated) and nullptr. Remember that svm.cpp is a library that we replicate. I think that the less we touch the better